src/pheromones/hivemind-host/include/hivemind-host/IpDiscoveryDTO.h

```diff
@@ -0,0 +1,51 @@
+#ifndef PROPOLIS_IPDISCOVERYDTO_H
+#define PROPOLIS_IPDISCOVERYDTO_H
+
+#include <cstdint>
+#include <message.pb.h>
+#include <variant>
+
+/**
+ * @brief Class to represent an IP request
+ */
+class IPDiscoveryDTO {
+
+  public:
+    IPDiscoveryDTO(const IPDiscovery & ipDiscovery);
+
+    IPDiscoveryDTO(uint32_t ipAddress);
+
+    /**
+    *@brief default constructor, the ip will be initialized as a monostate (ie, no ip)
+    */
+    IPDiscoveryDTO();
```

Comment:
I'm curious, when is this used? when do we want to create an object with an invalid state?

Reply:
Unused, I'll remove it